Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM debian:stable-slim
 
+ARG USERNAME=devops
+ARG USER_UID=1000
```

Comment:
When I was testing locally, it only worked when I used this UID, instead of the 2000 used before.